src/Microsoft.OData.Edm/Vocabularies/Annotations/EdmVocabularyAnnotation.cs

```diff
@@ -44,8 +47,67 @@ public EdmVocabularyAnnotation(IEdmVocabularyAnnotatable target, IEdmTerm term,
             this.term = term;
             this.qualifier = qualifier;
             this.value = value;
+            this.usesDefault = false;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the EdmVocabularyAnnotation class
+        /// to the default value.
+        /// </summary>
+        /// <param name=“target”>Element the annotation applies to.</param>
+        /// <param name=“term”>Term bound by the annotation</param>
+        public EdmVocabularyAnnotation(IEdmVocabularyAnnotatable target, IEdmTerm term)
+            :this(target, term, "")
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the EdmVocabularyAnnotation class 
+        /// to the default value.
+        /// </summary>
+        /// <param name=“target”>Element the annotation applies to.</param>
+        /// <param name=“term”>Term bound by the annotation</param>
+        /// <param name=“qualifier”>Qualifier used to distinguish bindings.</param>
+        public EdmVocabularyAnnotation(IEdmVocabularyAnnotatable target, IEdmTerm term,
+        string qualifier)
+        {
+            // Check arguments
+            EdmUtil.CheckArgumentNull(target, "target");
+            EdmUtil.CheckArgumentNull(term, "term");
+            if (term.DefaultValue == null)
+            {
+                throw new InvalidOperationException(Strings.EdmVocabularyAnnotations_DidNotFindDefaultValue(term.Name));
+            }
+
+            // Initialize
+            this.target = target;
+            this.term = term;
+            if (qualifier.Length > 0)
+            {
+                this.qualifier = qualifier;
+            } else
+            {
+                this.qualifier = null;
+            }
+            IEdmExpression valueExpression = EdmTermExtensions.BuildDefaultEdmExpression(term.Type, term.DefaultValue);
+            if (valueExpression == null)
+            {
+                throw Error.NotSupported();
+            }
+            this.value = valueExpression;
+            this.usesDefault = true;
+            //initialize(target, term, qualifier, value);
         }
 
+        /*private void initialize(IEdmVocabularyAnnotatable target, IEdmTerm term, string qualifier, IEdmExpression value)
+        {
+            this.target = target;
+            this.term = term;
+            this.qualifier = qualifier;
+            this.value = value;
+            this.usesDefault = true;
+        }*/
+

```

Comment:
you can remove these lines

Reply:
I think this might be from a previous version -- it's now removed